### PR TITLE
Readme for Segment_Tree_RMQ

### LINKED ISSUE #2164: Segment Tree Range Minimum Query
- [x] C - @MdAkdas
- [x] C++
- [ ] C#
- [ ] CoffeeScript
- [x] Dart - @Rukmini-Meda
- [ ] Go
- [x] Java
- [ ] JS - @JanaSabuj
- [ ] Kotlin
- [ ] Php
- [x] Python
- [x] Readme - @chandrikadeb7
- [ ] Ruby
- [ ] Rust
- [ ] TypeScript

### PR DESCRIPTION
**Pull request by chandrikadeb7**
Fixes #2164 

#### Checklist:
- [ ] 4 space indentation.
- [ ] Coding conventions are followed.
- [ ] Input is taken dynamically.
- [ ] Sample Input / Output is added at the end of file.
- [ ] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Added Readme.md for Segment_Tree_RMQ
- 

#### Languages Used:
- 
- 

#### Files Added:
- Segment_Tree_RMQ/Readme.md
- 

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

File: Segment_Tree_RMQ/README.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+```
+
+## Pseudocode
+
+After we've built a tree, we want to be able to search the tree for its values. (Which, in this case, is the minimum value between indices L and R.) 
+
+Similar to the technique we used before, we recursively search the tree, starting from the root, and if a node is within the range [L, R], then we check it's value as a valid minimum of the range. 
+
+```
+function RecursivelySearchForMin(L, R, nodeIndex, nodeStartIndex, nodeEndIndex)
```
Comment (singh-shreya6) [inline Segment_Tree_RMQ/README.md:52]: Please follow the indentation format for code here also. The comments can be changed to multi  line comments.

File: Segment_Tree_RMQ/README.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+  // really big number (which wouldn't be taken as the minimum).
+  if(nodeEndIndex < L || R < nodeStartIndex)
+    { 
+    return reallyBigNumber;
+    }
+  
+  // If this node is completely within [L, R], 
+  // then return it as the min value.
+  if(L <= nodeStartIndex && nodeEndIndex <= R)
+    {
```
Comment (singh-shreya6) [inline Segment_Tree_RMQ/README.md:65]: Please add proper indentation for braces and return statement should have 4 space indentation

**Comment by chandrikadeb7** [comment]
> Looks good. Please squash your commits.

Done!

**Comment by chandrikadeb7** [comment]
@singh-shreya6 Please merge my PR and assign the labels.

**Pull request merged**